source/index.html

```diff
@@ -133,49 +153,163 @@ <h2 class="visually-hidden">
-                    <p class="rates__cost"><span class="visually-hidden">Стоимость тарифного плана</span>9,99 usd</p>
+                    <p class="rates__name">
+                      <span class="visually-hidden">
+                        Название тарифного плана
+                      </span>
+                      Анлим
+                    </p>
+                    <p class="rates__cost">
+                      <span class="visually-hidden">
+                        Стоимость тарифного плана
+                      </span>
+                      9,99 usd
+                    </p>
                   </th>
                 </tr>
               </thead>
               <tbody>
                 <tr>
-                  <td class="rates__advantage"><span class="visually-hidden">Вид преимущества</span>Розовый фильтр<span class="visually-hidden">включено в тариф</span></td>
-                  <td class="rates__advantage"><span class="visually-hidden">Вид преимущества</span>Розовый фильтр<span class="visually-hidden">включено в тариф</span></td>
-                  <td class="rates__advantage"><span class="visually-hidden">Вид преимущества</span>Розовый фильтр<span class="visually-hidden">включено в тариф</span></td>
+                  <td class="rates__advantage rates__advantage--active ">
+                    <span class="visually-hidden">
+                      Вид преимущества
+                    </span>
+                    Розовый фильтр
+                    <span class="visually-hidden">
+                      включено в тариф
+                    </span>
+                  </td>
+                  <td class="rates__advantage rates__advantage--active">
+                    <span class="visually-hidden">
+                      Вид преимущества
+                    </span>
+                    Розовый фильтр
+                    <span class="visually-hidden">
+                      включено в тариф
+                    </span>
+                  </td>
+                  <td class="rates__advantage rates__advantage--active">
+                    <span class="visually-hidden">
+                      Вид преимущества
+                    </span>
+                    Розовый фильтр
+                    <span class="visually-hidden">
+                      включено в тариф
+                    </span>
+                  </td>
                 </tr>
                 <tr>
-                  <td class="rates__advantage"><span class="visually-hidden">Вид преимущества</span>Смайлики<span class="visually-hidden">не включено в тариф</span></td>
-                  <td class="rates__advantage"><span class="visually-hidden">Вид преимущества</span>Смайлики<span class="visually-hidden">включено в тариф</span></td>
-                  <td class="rates__advantage"><span class="visually-hidden">Вид преимущества</span>Смайлики<span class="visually-hidden">включено в тариф</span></td>
+                  <td class="rates__advantage rates__advantage--non-active">
```

Comment:
Какое-то странное слово...

```suggestion
                  <td class="rates__advantage rates__advantage--inactive">
```

Reply:
поменял